src/screens/ExchangeModal.js

```diff
@@ -143,7 +143,7 @@ class ExchangeModal extends Component {
       this.lastFocusedInput === null
     ) {
       this.inputFocusInteractionHandle = InteractionManager.runAfterInteractions(
-        this.inputFieldRef.focus
+        this.inputFieldRef.focus()
```

Comment:
This is to make sure the inputFieldRef actually focuses on initial load (it wasn't, before)